warp/src/raygun/mod.rs

```diff
@@ -548,41 +548,54 @@ impl Default for ConversationSettings {
 #[repr(C)]
 pub struct DirectConversationSettings {}
 
-#[derive(Default, Debug, Copy, Clone, Serialize, Deserialize, PartialEq, Eq, Hash, Display)]
-#[display(
-    fmt = "Everyone can add participants: {}, Everyone can change name: {}",
-    "if self.members_can_add_participants {\"✅\"} else {\"❌\"}",
-    "if self.members_can_change_name {\"✅\"} else {\"❌\"}"
-)]
+#[derive(Default, Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Display)]
+#[display(fmt = "Roles: {:?}", "self.roles")]
 #[repr(C)]
 pub struct GroupSettings {
```

Comment:
We might want to eventually rename this

Reply:
removed `ConversationSettings`/`GroupSettings` in favour of `GroupPermissions`